### ISSUE TITLE
Paper Mario: The Thousand-Year Door - 0100ecd018ebe000

### ISSUE THREAD

**Issue by piplup55** [Ryujinx/Ryujinx-Games-List#4972]
### Game Name

Paper Mario: The Thousand-Year Door

### Game Update Version

1.0.0

### Emulator Build Version

1.1.1318

### Graphics Backend

Vulkan

### Hardware Specs

- ##### CPU: AMD Ryzen 9 7900X3D
- ##### GPU: AMD RX 6600 XT / NVIDIA 3060
- ##### GPU Driver version: Mesa 24.0.7-arch1.3
- ##### RAM: 64gb

### Game State

Game has graphical issues on amd and nvidia and sometimes crashess on boot up

### Screenshots

![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/100526773/cb4ce6d8-28fe-45ed-aec1-ca569a9e0125)
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/100526773/da4821b7-1268-4732-9dea-5e510a2d0c2d)
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/100526773/0584c33c-de38-47e0-bcd2-ecdc5e88e999)

### Log File

[Ryujinx_1.1.1318_2024-05-22_15-47-50.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15405339/Ryujinx_1.1.1318_2024-05-22_15-47-50.log)

**Comment by Xmanant03real** [comment]
Already! sweet

**Comment by plexecutor** [comment]
Game does not launch on Steam Deck. Launches fine on Windows with an AMD CPU and Nvidia GPU. Latest releases.

**Comment by Nicalay2** [comment]
> Game does not launch on Steam Deck. Launches fine on Windows with an AMD CPU and Nvidia GPU. Latest releases.

For me the game launches, but the underground is fully black.
Also there are a few visuals glitches too.

**Comment by plexecutor** [comment]
Tested on Steam Deck using OpenGL and it launches. Does not work with Vulkan on Steam Deck. @Nicalay2 not sure why you down voted me, I was not disagreeing with your findings. Just adding my own experience.

**Comment by Nicalay2** [comment]
>@Nicalay2 not sure why you down voted me, I was not disagreeing with your findings. Just adding my own experience.

@plexecutor afaik, I'm not jeffreyvdhondel, i'm not the guy who downvoted you and I have no intention to do so.

> Tested on Steam Deck using OpenGL and it launches. Does not work with Vulkan on Steam Deck. 

Weird, my (short) experience on the Deck is with Vulkan, I didn't try with OpenGL. Now i'm playing on my PC since the underground isn't fully black on it, but I may try OpenGL.

**Comment by jeffreyvdhondel** [comment]
I have down voted because i'm not having the issue with my steam deck :)
Didn't wanted to comment on this to not go offtopic.

**Comment by Lolly-hub** [comment]
The game is not launching on my macbook. Please help

**Comment by plexecutor** [comment]
> I have down voted because i'm not having the issue with my steam deck :) Didn't wanted to comment on this to not go offtopic.

This is the Games List issue github. The instructions say to look for a game before creating a new issue which is what I did. There is nothing off topic in my post since it is related to this game.

- Search for a game first before adding a new issue; if you find an existing issue that is out of date, reply to it with a copied/pasted new issue template. A curator will then edit the OP with your test data.

**Comment by kylian562** [comment]
I got same issue and manage to go on pipe, everything go fine until after put sun and moon key

**Comment by Ethanator611** [comment]
I'm a victim of it crashing upon boot
https://github.com/Ryujinx/Ryujinx-Games-List/assets/109449034/c0e4041a-d6a8-4de3-b4a8-2c236123b49a

**Comment by Icarus170303** [comment]
Haven't run into any crashes, there are multiple textures/objects that are completely black. (An example in the image, another example is all of the crystal stars)
Backend: Vulkan
GPU: NVidia GTX 1660Ti

![Screenshot 2024-05-23 024146](https://github.com/Ryujinx/Ryujinx-Games-List/assets/40916331/0c855a9d-f94c-4d6a-9591-805c7cf0f7a5)

**Comment by Greyrat0** [comment]
Anyone know when the AMD black screen will be fixed? I've tried everything to get past this screen but the moving platform to get to the other pipe is impossible without seeing anything.

**Comment by TripleFuture** [comment]
I have also run into the black texture issue for save blocks, coins and crystal stars and I have an Nvidia GPU.

Cpu: ryzen 7 2700x
Gpu: rtx 4070

**Comment by Max833** [comment]
The ships flipping animation at the beginning is broken as well.

**Comment by Oliver-makes-code** [comment]
Some paper animations, i.e. the boat flip and bridge building in petal meadows animations, are broken. The boat flip shows the boat as invisible, while the bridge animation shows a black screen until the animation completes.

**Comment by Vbuck-gang** [comment]
black screen in sewer is fixed on amd as of latest build
![Uploading Screenshot (128).png…]()

**Comment by Lightspine** [comment]
Also, the crystal stars are not rendered properly. This includes when Sweet Treat is used and when the crystal star is collected after defeating the first chapter's boss.

**Comment by Nicalay2** [comment]
![20240523202359_1](https://github.com/Ryujinx/Ryujinx-Games-List/assets/117358978/10bde39d-c57a-4218-9a64-534a52dced51)
![20240523200115_1](https://github.com/Ryujinx/Ryujinx-Games-List/assets/117358978/746740ce-e13e-41c2-8f2f-d80fbe481ffa)
Here are the screenshots.

**Comment by RupeeClock** [comment]
> Some paper animations, i.e. the boat flip and bridge building in petal meadows animations, are broken. The boat flip shows the boat as invisible, while the bridge animation shows a black screen until the animation completes.

Providing evidence of the boat flip animation missing:

### Video clip
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/c5907690-be2f-4da6-aa1a-848755c56aa4

### Machine specs
**CPU** - Intel Core i7-12700k
**GPU** - Nvidia RTX 3070 8GB - driver version 552.44
**RAM** - 32GB DDR5 

### Ryujinx configs
**Ryujinx version** - 1.1.1320
**Video renderer** - Vulkan

### Mods loaded
* Config 1: None
* Config 2: Assorted from https://gamebanana.com/mods/cats/29423?
    * [60 FPS v1.0.0]
    * [1920x1080 v1.0.0]
    * [Level of Detail v1.0.0]
    * Reencoded video

### Log file
[Ryujinx_1.1.1320_2024-05-23_20-47-46.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15422052/Ryujinx_1.1.1320_2024-05-23_20-47-46.log)

**Comment by reflectronic** [comment]
Some shiny surfaces produce glitchy artifacts:

https://github.com/Ryujinx/Ryujinx-Games-List/assets/27514983/dd1af172-615c-459e-b86f-e0aaab47b85f


CPU: i7-4770
GPU: Radeon 6600, Driver 24.5.1
RAM: 16 GB
Vulkan
Ryujinx 1.1.1320 + https://github.com/Ryujinx/Ryujinx/pull/6853 applied

**Comment by Oliver-makes-code** [comment]
> Some paper animations, i.e. the boat flip and bridge building in petal meadows animations, are broken. The boat flip shows the boat as invisible, while the bridge animation shows a black screen until the animation completes.

This includes the staircase animation in hooktail's castle

https://github.com/Ryujinx/Ryujinx-Games-List/assets/59418903/12e000e8-4078-415c-b2ec-854a54a7fe3a

**Comment by kobayashi90** [comment]
Basically all shiny surfaces, eg. glass doors in Chapter 1,2,3 & 7 and also in 8

**Comment by Nitsu29** [comment]
Sometimes the Music has less volume after i pick up an item or Shine Sprite or when i do something that supposed to lower the volume for a short period of time like entering dialogue. 
The only way to fix it is when the Music changes (with the badge that changes it to the OG Tracks or when i enter an area with another music track). 
Changing the area with the same music track does not fix it.

**Comment by mrhdog13** [comment]
Save block and Crystal shards are still black for me and I am on the latest release 1.1.1321 I use vulkan Nivdia gpu pls fix

**Comment by luminouslily35** [comment]
@Ethanator611 you're not the only one!



![WindowsTerminal_j7erC3HxJc](https://github.com/Ryujinx/Ryujinx-Games-List/assets/97817095/d1c1bc63-3843-4860-b3f2-81f67dfbbee7)

[Ryujinx_1.1.1322_2024-05-25_01-02-21.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15442225/Ryujinx_1.1.1322_2024-05-25_01-02-21.log)

**Comment by RupeeClock** [comment]
> @Ethanator611 
> [Ryujinx_1.1.1322_2024-05-25_01-02-21.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15442225/Ryujinx_1.1.1322_2024-05-25_01-02-21.log)

These logs indicate you have mods loaded, try booting the game with no mods loaded first and see if it works.

**Comment by sallaboy** [comment]
The game crashes on the lunch on 
AMD Radeon RX 6600 XT
and also on
Rog ALLY

also with the latest version 1322
![Screenshot 2024-05-25 140632](https://github.com/Ryujinx/Ryujinx-Games-List/assets/2448836/55b378df-4389-402d-bffb-29dc7ba046fa)

[Ryujinx_1.1.1322_2024-05-25_14-05-51.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15443143/Ryujinx_1.1.1322_2024-05-25_14-05-51.log)

I couldn't find any way to run it. Please help.

**Comment by KayLilac** [comment]
seems like ssr and bloom aren't working on intel (at least on vulkan, haven't tried opengl because my emulator crashes)

![Paper Mario_ The Thousand-Year Door_2024-05-25_12-47-39](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/ea121e12-d6ec-48cf-867b-2bfe18e2b309)

there might be other gfx that don't work but these are the one's i noticed
if needed i can send a log file

Ryujinx version: 1.1.1322
GPU: Intel Iris Xe (integrated)
RAM: 24gb
CPU: 11th Gen Intel Core i5-1135G7

**Comment by KayLilac** [comment]
> seems like ssr and bloom aren't working on intel (at least on vulkan, haven't tried opengl because my emulator crashes)
> 
> ![Paper Mario_ The Thousand-Year Door_2024-05-25_12-47-39](https://private-user-images.githubusercontent.com/149713452/333814273-ea121e12-d6ec-48cf-867b-2bfe18e2b309.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2NTMwOTIsIm5iZiI6MTcxNjY1Mjc5MiwicGF0aCI6Ii8xNDk3MTM0NTIvMzMzODE0MjczLWVhMTIxZTEyLWQ2ZWMtNDhjZi04NjdiLTJiZmUxOGUyYjMwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNVQxNTU5NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iZjQzNGNhOTNjMmNjY2RjMDUyMDU3YzgwMzZlNWNhOTQ2ODdmZDVlOTUzZmNkNjYxNDgzNGQxNjA1NDNkNjA1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.sClxrwzfDdVvx0poi0UAfcdzxtHk1hfadQr0LcExKhs)
> 
> there might be other gfx that don't work but these are the one's i noticed if needed i can send a log file
> 
> Ryujinx version: 1.1.1322 GPU: Intel Iris Xe (integrated) RAM: 24gb CPU: 11th Gen Intel Core i5-1135G7

forgot to mention: when loading a new area the ssr do appear but only for a few seconds then disappear

**Comment by GamerzHell9137** [comment]
> seems like ssr and bloom aren't working on intel (at least on vulkan, haven't tried opengl because my emulator crashes)
> 
> ![Paper Mario_ The Thousand-Year Door_2024-05-25_12-47-39](https://private-user-images.githubusercontent.com/149713452/333814273-ea121e12-d6ec-48cf-867b-2bfe18e2b309.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2Njk4MjIsIm5iZiI6MTcxNjY2OTUyMiwicGF0aCI6Ii8xNDk3MTM0NTIvMzMzODE0MjczLWVhMTIxZTEyLWQ2ZWMtNDhjZi04NjdiLTJiZmUxOGUyYjMwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNVQyMDM4NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMTE5NjdjYjkwOWJlNmIyNjc2YTIwYjQ2YTM0ZTgxNzA1Nzg5ZWZiMTE3OGE4NWY4NDMyOWFkMDU0NzYwYmU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.8gm311x5OknUcTuA09-jYiqeIR98a61clAlOGIvWD5w)
> 
> there might be other gfx that don't work but these are the one's i noticed if needed i can send a log file
> 
> Ryujinx version: 1.1.1322 GPU: Intel Iris Xe (integrated) RAM: 24gb CPU: 11th Gen Intel Core i5-1135G7

Always post logs with issues.

**Comment by KayLilac** [comment]
> > seems like ssr and bloom aren't working on intel (at least on vulkan, haven't tried opengl because my emulator crashes)
> > ![Paper Mario_ The Thousand-Year Door_2024-05-25_12-47-39](https://private-user-images.githubusercontent.com/149713452/333814273-ea121e12-d6ec-48cf-867b-2bfe18e2b309.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2Njk4MjIsIm5iZiI6MTcxNjY2OTUyMiwicGF0aCI6Ii8xNDk3MTM0NTIvMzMzODE0MjczLWVhMTIxZTEyLWQ2ZWMtNDhjZi04NjdiLTJiZmUxOGUyYjMwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyNVQyMDM4NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMTE5NjdjYjkwOWJlNmIyNjc2YTIwYjQ2YTM0ZTgxNzA1Nzg5ZWZiMTE3OGE4NWY4NDMyOWFkMDU0NzYwYmU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.8gm311x5OknUcTuA09-jYiqeIR98a61clAlOGIvWD5w)
> > there might be other gfx that don't work but these are the one's i noticed if needed i can send a log file
> > Ryujinx version: 1.1.1322 GPU: Intel Iris Xe (integrated) RAM: 24gb CPU: 11th Gen Intel Core i5-1135G7
> 
> Always post logs with issues.

ok! here it is
[Ryujinx_1.1.1322_2024-05-25_21-18-04.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15445589/Ryujinx_1.1.1322_2024-05-25_21-18-04.log)
i'm kinda new to this, my apologies

**Comment by User06192090** [comment]
can someone help me how to download it?

**Comment by RupeeClock** [comment]
Have confirmed the following fixed in Ryujinx v1.1.1325:

* Coloration of reflective surface on spinning coin in the Prologue intro
* Rainbow coloration of save block
* Boat "paper flip" animation

The game looks and plays exceptionally well on my machine with my chosen mods, rendering to a 3840x2160 60hz display with 7680x4320 internal resolution. Have not played beyond gaining player control at this stage to see if there any performance impacts with these very high settings, issues with mods usage, or yet unencountered but previously identified inaccuracies.

### Evidence
https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/177a9532-fba5-425e-a373-23f630a01910
![Paper Mario_ The Thousand-Year Door_2024-05-26_21-17-20](https://github.com/Ryujinx/Ryujinx-Games-List/assets/24753760/ae946e2e-6af3-4027-8c5b-ea9d1b8f6f5d)

### Machine specs
**CPU** - Intel Core i7-12700k
**GPU** - Nvidia RTX 3070 8GB - driver version 552.44
**RAM** - 32GB DDR5 

### Ryujinx configs
**Ryujinx version** - 1.1.1325
**Video renderer** - Vulkan

### Mods loaded
* Config 2: Assorted from https://gamebanana.com/mods/cats/29423
    * [60 FPS v1.0.0]
    * [3840x2160 v1.0.0]
    * [Level of Detail v1.0.0]
    * [Disable AA]
    * Reencoded video

### Log file
[Ryujinx_1.1.1325_2024-05-26_21-20-16.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15448858/Ryujinx_1.1.1325_2024-05-26_21-20-16.log)

**Comment by KayLilac** [comment]
uh..
game looks very broken on intel in the new ryujinx version 
![Paper Mario_ The Thousand-Year Door_2024-05-26_18-28-34](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/469e8d35-9320-4953-b8b2-23945cfd18f5)
![Paper Mario_ The Thousand-Year Door_2024-05-26_18-29-05](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/e38e6f60-91bd-40b3-b892-3e6425f4935d)
![Paper Mario_ The Thousand-Year Door_2024-05-26_18-29-32](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/4678130c-3ee0-4ef6-8a85-7819d1690e61)

log file:
[Ryujinx_1.1.1325_2024-05-26_18-25-14.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15449011/Ryujinx_1.1.1325_2024-05-26_18-25-14.log)

**Comment by silvergit88** [comment]
Hello, for me game doesnt run, it loads but before the nintendo logo emulator closes.

Machine specs
CPU - AMD Ryzen 5 3600XT
GPU - AMD Radeon RX 6750 XT | Adrenalin Edition 24.5.1 | Driver version 23.40.33.01
RAM - 32GB DDR4

Ryujinx version - 1.1.1325
System Version 18.0.0
Video renderer - Vulkan

Mods loaded
Config 1: None

Log file
[Ryujinx_1.1.1325_2024-05-26_17-27-13.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15449391/Ryujinx_1.1.1325_2024-05-26_17-27-13.log)

**Comment by cannon9009** [comment]
There appears to be an issue with audio where sounds begin to trail behind what's visually happening on screen. I haven't been able to pinpoint what exactly causes this; it just seems to randomly happen at some point.
[Ryujinx_1.1.1325_2024-05-26_18-09-09.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15449601/Ryujinx_1.1.1325_2024-05-26_18-09-09.log)
[Ryujinx_1.1.1325_2024-05-26_16-18-29.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15449603/Ryujinx_1.1.1325_2024-05-26_16-18-29.log)
[Ryujinx_1.1.1322_2024-05-26_16-18-12.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15449604/Ryujinx_1.1.1322_2024-05-26_16-18-12.log)
Hopefully these logs are of some use to you all.

**Comment by Jamillian1** [comment]
Game crashes when trying to go down to sublevel one on the X-Naut Base/Moon on MacBook Pro M1 Max (16 GB Ram)

**Comment by ChiliEater** [comment]
In my experience, going through loading zones causes the audio to slightly drift. This effect accumulates pretty fast and results in a pretty jarring experience.

https://github.com/Ryujinx/Ryujinx-Games-List/assets/32131318/4b87b5f5-74d0-48ad-9e92-af8207baffc1

https://github.com/Ryujinx/Ryujinx-Games-List/assets/32131318/6afc5f21-0a06-448d-9923-8b21168c2e25

The only log entries that I could find all looked like this:

```log
Mai 27 13:19:22 batpc-arch plasmashell[5681]: 00:03:05.603 |E| HLE.OsThread.4 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 17157 took more than 1000ms, resuming execution...
Mai 27 13:19:45 batpc-arch plasmashell[5681]: 00:03:28.183 |W| HLE.OsThread.4 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
```

Config:
- Ryujinx 1.1.1325+c41fddd25e8ddd3cf0b3cefeaf6595d2e4ede0fa
- Vulkan
- SDL2
- Mods
  - Level of Detail
  - Disable AA

Specs:
- Arch Linux
- KDE Plasma (Wayland)
- Using Pipewire + Wireplumber for audio
- NVIDIA RTX 2070 (555.42.02)
- Ryzen 7 2700X

**Comment by Djipi71** [comment]
For me even the update , i have some black texture on windows.
Build 1.1.1325

![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/35375131/c104e3c8-6359-431d-9aff-a587a6a62a48)
look at the key boat...

**Comment by smokeytm** [comment]
I've been getting these weird artifacts on the textures that used to be missing before
![pmttyd](https://github.com/Ryujinx/Ryujinx-Games-List/assets/170573128/157f1d55-5f4d-4a2e-8e71-866c10945689)
![pmttyd1](https://github.com/Ryujinx/Ryujinx-Games-List/assets/170573128/38b15f60-9f3d-444b-8c33-8c38829f4bce)

Machine specs
CPU - Intel i3-10105F
GPU - AMD Radeon RX 5700xt
RAM - 8GB DDR4

Ryujinx version - 1.1.1325
System Version 18.0.0
Video renderer - Vulkan

no mods

Log file
[Ryujinx_1.1.1325_2024-05-27_16-19-54.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15459350/Ryujinx_1.1.1325_2024-05-27_16-19-54.log)

**Comment by bkcdm** [comment]
how can i download it

**Comment by Phantoful** [comment]
> Sometimes the Music has less volume after i pick up an item or Shine Sprite or when i do something that supposed to lower the volume for a short period of time like entering dialogue. The only way to fix it is when the Music changes (with the badge that changes it to the OG Tracks or when i enter an area with another music track). Changing the area with the same music track does not fix it. i'm using SoundIO, but it also occurs with SDL2 if that helps
> R.05-25.00-28.3.mp4
> 
> Looks like skipping through the animation when i get an item causes the problem. When i get the Suitcase and let it play out the music returns to normal after that, however the music stays more quiet when i skip though the animation of the other two

I get the same issue

**Comment by riperiperi** [comment]
For audio delay issues, it's generally a problem with SDL2 audio on Linux. For block flickering on certain textures, it's an AMD issue.

**Comment by cannon9009** [comment]
> For audio delay issues, it's generally a problem with SDL2 audio on Linux.
Issue occurs on Windows too. I probably should have mentioned that when I reported the issue; that is my fault and I apologize.

**Comment by AbnormalPoof** [comment]
Game crash after attempting to use the first elevator in the X-Naut base, rendering the game unbeatable for now. Graphical artifacts appear for a little bit right before crashing.

Specs
- CPU: Apple M1
- GPU: Apple M1
- RAM: 8 GB
- macOS Sonoma 14.5

Ryujinx:
- Version 1.1.1325
- Vulkan renderer
- No mods enabled

Log file:
[Ryujinx_1.1.1325_2024-05-28_02-02-48.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15465289/Ryujinx_1.1.1325_2024-05-28_02-02-48.log)

![Screenshot 2024-05-28 at 2 04 06 AM](https://github.com/Ryujinx/Ryujinx-Games-List/assets/86753001/ce56538e-2f67-4556-ad7c-0cbb6a82de11)

**Comment by Vbuck-gang** [comment]
Text bubble corruption
![Screenshot (135)](https://github.com/Ryujinx/Ryujinx-Games-List/assets/63995419/2789eebb-5b01-4c35-bb2b-f717c43ce84b)

System:
GPU-7900xt
CPU-5900x
Ram-32 gigs of Crucial bullistix 3200mhz
SSD- 970 EVO plus

Vulkan set as renderer

60Fps mod on
Logs:
[Ryujinx_1.1.1325_2024-05-28_12-37-18.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15476726/Ryujinx_1.1.1325_2024-05-28_12-37-18.log)

**Comment by negrinni2064** [comment]
Dodgy status effect of Macho Grubba seems to be bugged, as it's not properly displayed and the model becomes invisible. The correct behaviour of this effect should be like the following: https://www.youtube.com/watch?v=RV7AOhxIN9I (minute 9:00; sorry for the unintended spam).

Initially I thought it was a mod-related issue but when I disabled the mods and tried again the issue persisted, so here I am.

In the attached clip of ths post I provide you the detail of the issue and also the log file of the game session.

Specs:

CPU: AMD Ryzen 7 5800X
GPU: EVGA GeForce RTX 3060 Ti FTW3 ULTRA GAMING
RAM: Kingston Fury Renegade DDR4-3600 (2x16)
SSD: WD SN770 NVMe 500 GB

Ryujinx Version: 1.1.1325
Windows OS: 10
Firmware Version: 18.0.0
All the plethora of mods disabled

[Ryujinx_1.1.1325_2024-05-28_17-55-23.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15477271/Ryujinx_1.1.1325_2024-05-28_17-55-23.log)

https://github.com/Ryujinx/Ryujinx-Games-List/assets/171093162/0001cb3e-5354-43c9-b869-0eee4d3f49c7

**Comment by Vbuck-gang** [comment]
> Dodgy status effect of Macho Grubba seems to be bugged, as it's not properly displayed and the model becomes invisible. The correct behaviour of this effect should be like the following: https://www.youtube.com/watch?v=RV7AOhxIN9I (minute 9:00; sorry for the unintended spam).
> 
> Initially I thought it was a mod-related issue but when I disabled the mods and tried again the issue persisted, so here I am.
> 
> In the attached clip of ths post I provide you the detail of the issue and also the log file of the game session.
> 
> Specs:
> 
> CPU: AMD Ryzen 7 5800X GPU: EVGA GeForce RTX 3060 Ti FTW3 ULTRA GAMING RAM: Kingston Fury Renegade DDR4-3600 (2x16) SSD: WD SN770 NVMe 500 GB
> 
> Ryujinx Version: 1.1.1325 Windows OS: 10 Firmware Version: 18.0.0 All the plethora of mods disabled
> 
> [Ryujinx_1.1.1325_2024-05-28_17-55-23.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/15477271/Ryujinx_1.1.1325_2024-05-28_17-55-23.log)
> 
>  Ryujinx.Nvidia.Profile.2024.05.28.-.17.57.12.05.mp4

this happens with all characters that have the repel effect on them. i thought it was the intended effect because it's an item that significantly increases dodge chance but from that switch gameplay you showed i see that it's clearly not the case.

https://github.com/Ryujinx/Ryujinx-Games-List/assets/63995419/e552d179-79ba-4721-8aa0-b9cd104bc299

**Comment by Mauhid123qw** [comment]
Mario moves slowly for some reason, even on Yoshi. As you can see, it can cause problems in some areas. This did not originally occur earlier in my playthrough, only happened recently for some reason.

GPU: Apple M2
Graphics Backend: Vulkan
Hypervisor: off
Memory Manager Mode: Host (fast)
RAM: 8 GB

Version: 1.1.1325
No mods enabled

[Video](https://drive.google.com/file/d/1NGGZZLkcqFCXFasCf82cIDzmvcJk0JfS/view?usp=sharing)

**Comment by teogorghin** [comment]
> Game loads forever when entering the temple in the painting in chapter 6
> 
> GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> 
> [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> 
>  2024-05-31.20-42-32.mp4

Same issue on Intel Iris Xe (both OpenGL and Vulkan).
There is any workaround?
It also softlocks on Sudachi (both OpenGL and Vulkan).

**Comment by KayLilac** [comment]
as of ryujinx 1.1.1330, all of the issues on intel i previously reported have been fixed! nice work y'all!
![Paper Mario_ The Thousand-Year Door_2024-06-03_17-16-46](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/74b5f93b-5d81-40c2-a71e-6ccd0c3f85da)
![Paper Mario_ The Thousand-Year Door_2024-06-03_17-05-09](https://github.com/Ryujinx/Ryujinx-Games-List/assets/149713452/05acc50a-e5a9-4275-b7ad-6b4353aca469)

**Comment by antoniodesousa** [comment]
Game crashes immediately after booting.
[Ryujinx_1.1.1330_2024-06-05_18-05-03.log](https://github.com/user-attachments/files/15593411/Ryujinx_1.1.1330_2024-06-05_18-05-03.log)

CPU: Ryzen 9 6900HX
GPU: AMD RX 6600 XT
RAM: 64 GB DDR5
OS: Windows 11 Pro 23H2

**Comment by RupeeClock** [comment]
> @antoniodesousa 
> Game crashes immediately after booting. [Ryujinx_1.1.1330_2024-06-05_18-05-03.log](https://github.com/user-attachments/files/15593411/Ryujinx_1.1.1330_2024-06-05_18-05-03.log)

Logs indicate that you have mods loaded, try booting the game without mods and see if that works.
Issues with the mods can cause problems like invalid memory access either on boot or during gameplay.

**Comment by antoniodesousa** [comment]
> > @antoniodesousa
> > Game crashes immediately after booting. [Ryujinx_1.1.1330_2024-06-05_18-05-03.log](https://github.com/user-attachments/files/15593411/Ryujinx_1.1.1330_2024-06-05_18-05-03.log)
> 
> Logs indicate that you have mods loaded, try booting the game without mods and see if that works. Issues with the mods can cause problems like invalid memory access either on boot or during gameplay.

I don't have or use mods in any game.

![Ryujinx 1 1 1330 05_06_2024 19_14_11](https://github.com/Ryujinx/Ryujinx-Games-List/assets/10920974/36ab1476-c7e6-4f6f-b207-9f673da24985)

**Comment by antoniodesousa** [comment]
> > @antoniodesousa
> > Game crashes immediately after booting. [Ryujinx_1.1.1330_2024-06-05_18-05-03.log](https://github.com/user-attachments/files/15593411/Ryujinx_1.1.1330_2024-06-05_18-05-03.log)
> 
> Logs indicate that you have mods loaded, try booting the game without mods and see if that works. Issues with the mods can cause problems like invalid memory access either on boot or during gameplay.

I don't have or use mods in any game.

![Ryujinx 1 1 1330 05_06_2024 19_20_04](https://github.com/Ryujinx/Ryujinx-Games-List/assets/10920974/e7f0fe62-058e-4fd3-8b79-14f5f4b830b0)

**Comment by RupeeClock** [comment]
The mods you're using appear to be cheat codes.
```
00:00:01.030 |I| ModLoader LoadCheats: Build ids found for application 0100ECD018EBE000:
    7501ABFE55FA41CFFEB46BD619BDCBF45B9CAE3A000000000000000000000000
    78F37BB55D015BE3B368EC22AF595455F1544DC1000000000000000000000000
    368F01D5CB42F06CD125DC739D036719700CB459000000000000000000000000
    8618895246B420329B77746B960A66ABCBEDE992000000000000000000000000
```

**Comment by antoniodesousa** [comment]
@RupeeClock 
I haven't install or use any cheats ever... there most be something wrong with the logging.

![Ryujinx 1 1 1330 - Cheats Manager 05_06_2024 20_41_19](https://github.com/Ryujinx/Ryujinx-Games-List/assets/10920974/1818a256-e679-475d-b0c2-f57180038efb)

**Comment by SomethingOriginal** [comment]
I understand that increasing resolution may create unintended effects but this is a weird graphical glitch that fixes itself after using the sheet ability. This does not happen at native resolution, only once resolution is scaled up.
![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-33](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/44de47d4-db9f-4029-98d0-cafd8fa03efd)
![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-42](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/71be368d-1c29-4f15-b628-00f01865b62a)
![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-53](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/f22638af-2f33-425e-b216-44cd8854a6cf)
![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-59](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/d297d18c-eded-48e3-acc4-686ac4e509c8)
[Ryujinx_1.1.1330_2024-06-05_15-50-58.log](https://github.com/user-attachments/files/15595944/Ryujinx_1.1.1330_2024-06-05_15-50-58.log)

Hardware:
CPU - AMD 5800X3D
GPU - NVIDIA 3080 12GB, driver 551.52
RAM - 32GB DDR4
Ryujinx version - 1.1.1330
Video renderer - Vulkan
No mods, no cheats despite what the log claims.

**Comment by antoniodesousa** [comment]
@RupeeClock 
I don't know why but the game started working suddenly... I didn't change anything at all.
Only thing different from the last time I tried to play the game, it's that I didn't have any controllers connected so I got the warning popup, and afterwards the game started working. Anyways, consider my issue fixed.

**Comment by gameygu-0213** [comment]
> Game loads forever when entering the temple in the painting in chapter 6
> 
> GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> 
> [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> 2024-05-31.20-42-32.mp4

as of Ryujinx 1.1.1330, this still occurs. 

Steam Deck 256gb LCD Model

[Ryujinx_1.1.1330_2024-06-05_19-13-42.log](https://github.com/user-attachments/files/15597652/Ryujinx_1.1.1330_2024-06-05_19-13-42.log)

**Comment by ffrasisti** [comment]
Have the softlocking and Cape problems been fixed? Is it playable to the end now?

**Comment by Didgeridou** [comment]
> Game loads forever when entering the temple in the painting in chapter 6
> 
> GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> 
> [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> 
>  2024-05-31.20-42-32.mp4

Ok I figured out it was an issue with my game as I disabled fs integrity check. Now it's working really well.

**Comment by Didgeridou** [comment]
> Have the softlocking and Cape problems been fixed? Is it playable to the end now?

The softlock in chapter 6 might be due to your game being corrupted
The cape still makes enemies invisible as of 1.1.1330

**Comment by Vbuck-gang** [comment]
there is a visual glich in front of the chest in The Pit Of 100 Trials

https://github.com/Ryujinx/Ryujinx-Games-List/assets/63995419/e052b3f0-49c1-4e08-a57e-da894a508b62


System:
GPU-7900xt
CPU-5900x
Ram-32 gigs of Crucial bullistix 3200mhz
SSD- 970 EVO plus

Vulkan set as renderer

OS: Windows 10
60Fps mod on

**Comment by damage6** [comment]
Had a hard crash + reboot on Steam Deck. OLED 1TB model. Happened as I was going through a pipe transition - it played the entering effect, into black loading screen as usual, but then froze into hard crash / reboot. There are also black artifacts in the Pit of 100 Trials backdrop - next time I'm in I will take pictures and post the log. 
[Ryujinx_1.1.1330_2024-06-09_11-50-41.log](https://github.com/user-attachments/files/15754238/Ryujinx_1.1.1330_2024-06-09_11-50-41.log)

**Comment by PacMania67** [comment]
Crash. Have had a few of these happen before while playing, and they seem random in nature. This one was on the moon after a battle.

System:
GPU: NVIDIA RTX 4070
CPU: i7-14700k
RAM: Corsair Vengeance 64 GB (2 x 32 GB) DDR5-5200 CL40 Memory
SSD: Samsung 980 Pro 1 TB

Vulkan Renderer
OS: Windows 11 Pro

[Ryujinx_1.1.1330_2024-06-10_13-49-15.log](https://github.com/user-attachments/files/15777365/Ryujinx_1.1.1330_2024-06-10_13-49-15.log)

**Comment by calmexd** [comment]
I've encountered a crash every time I try to enter the elevator on the moon in Chapter 7. I'm playing on Mac.

**Comment by penguin21** [comment]
> > Game loads forever when entering the temple in the painting in chapter 6
> > GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> > [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> > 2024-05-31.20-42-32.mp4
> 
> Ok I figured out it was an issue with my game as I disabled fs integrity check. Now it's working really well.

Thats problem happends me too in Suyu (Yuzu) too!

**Comment by negativeiqtest** [comment]
> Game loads forever when entering the temple in the painting in chapter 6
> 
> GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> 
> [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> 
>  2024-05-31.20-42-32.mp4



> > Game loads forever when entering the temple in the painting in chapter 6
> > GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> > [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> > 2024-05-31.20-42-32.mp4
> 
> Same issue on Intel Iris Xe (both OpenGL and Vulkan). There is any workaround? It also softlocks on Sudachi (both OpenGL and Vulkan).



> > Game loads forever when entering the temple in the painting in chapter 6
> > GPU : NVIDIA RTX 4050 Laptop GPU CPU : AMD Ryzen 5 7640HS RAM : 16 GB DDR5 Graphics Backend : Vulkan Ryujinx version 1.1.1325 No mods enabled
> > [Ryujinx_1.1.1325_2024-05-31_20-47-06.log](https://github.com/user-attachments/files/15511805/Ryujinx_1.1.1325_2024-05-31_20-47-06.log)
> > 2024-05-31.20-42-32.mp4
> 
> as of Ryujinx 1.1.1330, this still occurs.
> 
> Steam Deck 256gb LCD Model
> 
> [Ryujinx_1.1.1330_2024-06-05_19-13-42.log](https://github.com/user-attachments/files/15597652/Ryujinx_1.1.1330_2024-06-05_19-13-42.log)

I am having the same issue.



> > Have the softlocking and Cape problems been fixed? Is it playable to the end now?
> 
> The softlock in chapter 6 might be due to your game being corrupted

Could you elaborate on this? You were the first one to report the issue

**Comment by MetrosexualGarbodor** [comment]
> Game loads forever when entering the temple in the painting in chapter 6

Given that both logs have FS integrity checks turned off, it's almost guaranteed to be a bad game dump.

**Comment by illegalmp3** [comment]
Consistent crash on macOS in Chapter 7 (Mario Shoots the Moon) upon entering an elevator and choosing a sublevel to go to. Once the choice is made, the elevator begins to move, and Ryujinx is instantly closed.

System:
GPU: Apple M1
CPU: Apple M1
RAM: 8 GB
SSD: 256 GB

Vulkan Renderer (default resolution, all graphics settings left on default)
DualShock 4 being used as controller
OS: macOS 15 Sequoia Developer Beta 1 (Please note that I experienced this same crash on macOS 14 Sonoma, using the same Ryujinx settings and same game dump/save files)

[Ryujinx_1.1.1336_2024-06-21_18-26-42.log](https://github.com/user-attachments/files/15934707/Ryujinx_1.1.1336_2024-06-21_18-26-42.log)

**Comment by ianrennie** [comment]
On windows Mario is weirdly aliased but on linux he is not. Both pictures were taken with no mods and rendered on Native switch output.
Windows 10: 
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/12806601/959c0a50-e79e-4cbd-831f-24a37c2d01ca)
Linux EndeavourOS: 
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/12806601/cf270944-3da6-4d4b-9129-2fabe3cee1e0)

Ryujinx version 1.1.1340
System:
GPU - 7900 GRE
CPU - 5800X
Ram - 64 gigs 3600mhz ripjaws

**Comment by zorglorfian** [comment]
> Consistent crash on macOS in Chapter 7 (Mario Shoots the Moon) upon entering an elevator and choosing a sublevel to go to. Once the choice is made, the elevator begins to move, and Ryujinx is instantly closed.
> 
> System: 2020 13-inch MacBook Pro GPU: Apple M1 CPU: Apple M1 RAM: 8 GB SSD: 256 GB
> 
> Vulkan Renderer (default resolution, all graphics settings left on default) DualShock 4 being used as controller OS: macOS 15 Sequoia Developer Beta 1 (Please note that I experienced this same crash on macOS 14 Sonoma, using the same Ryujinx settings and same game dump/save files)
> 
> [Ryujinx_1.1.1336_2024-06-21_18-26-42.log](https://github.com/user-attachments/files/15934707/Ryujinx_1.1.1336_2024-06-21_18-26-42.log)
Same with me. I hope this gets somehow fixed. Looking at comments from others on this issue, the entire Moon Base loads at once instead of separate floors, so it may be causing a strain on the computer that Mac can't handle. Others have also said that they have used Windows to get through Chapter 7, then copying their save back to Mac.

**Comment by OmniiOMEGA** [comment]
> In my experience, going through loading zones causes the audio to slightly drift. This effect accumulates pretty fast and results in a pretty jarring experience.
> 
> Update: Upon further testing, this only occurs on SDL2. OpenAL works just fine. SoundIO just crashes immediately and cuts my entire audio for a few seconds.
> 
>  ttyd-sync.mp4 
>  ttyd-desync.mp4 
> The only log entries that I could find all looked like this:
> 
> ```
> Mai 27 13:19:22 batpc-arch plasmashell[5681]: 00:03:05.603 |E| HLE.OsThread.4 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 17157 took more than 1000ms, resuming execution...
> Mai 27 13:19:45 batpc-arch plasmashell[5681]: 00:03:28.183 |W| HLE.OsThread.4 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
> ```
> 
> Config:
> 
> * Ryujinx 1.1.1325+c41fddd25e8ddd3cf0b3cefeaf6595d2e4ede0fa
> * Vulkan
> * SDL2
> * Mods
>   
>   * Level of Detail
>   * Disable AA
> 
> Specs:
> 
> * Arch Linux
> * KDE Plasma (Wayland)
> * Using Pipewire + Wireplumber for audio
> * NVIDIA RTX 2070 (555.42.02)
> * Ryzen 7 2700X

Same issue here...  my audio issues are crackle when going into paper modes...
After switching audio from SDL2 to OpenAL it appears the crackling stops? But I swear I tried that when the issue occurred and it didn't resolve...

Here are my logs...
[Ryujinx_1.1.1364_2024-08-02_17-47-32.zip](https://github.com/user-attachments/files/16468987/Ryujinx_1.1.1364_2024-08-02_17-47-32.zip)

**Comment by OmniiOMEGA** [comment]
> > In my experience, going through loading zones causes the audio to slightly drift. This effect accumulates pretty fast and results in a pretty jarring experience.
> > Update: Upon further testing, this only occurs on SDL2. OpenAL works just fine. SoundIO just crashes immediately and cuts my entire audio for a few seconds.
> > ttyd-sync.mp4
> > ttyd-desync.mp4
> > The only log entries that I could find all looked like this:
> > ```
> > Mai 27 13:19:22 batpc-arch plasmashell[5681]: 00:03:05.603 |E| HLE.OsThread.4 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 17157 took more than 1000ms, resuming execution...
> > Mai 27 13:19:45 batpc-arch plasmashell[5681]: 00:03:28.183 |W| HLE.OsThread.4 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Config:
> > 
> > * Ryujinx 1.1.1325+c41fddd25e8ddd3cf0b3cefeaf6595d2e4ede0fa
> > * Vulkan
> > * SDL2
> > * Mods
> >   
> >   * Level of Detail
> >   * Disable AA
> > 
> > Specs:
> > 
> > * Arch Linux
> > * KDE Plasma (Wayland)
> > * Using Pipewire + Wireplumber for audio
> > * NVIDIA RTX 2070 (555.42.02)
> > * Ryzen 7 2700X
> 
> Same issue here... my audio issues are crackle when going into paper modes... After switching audio from SDL2 to OpenAL it appears the crackling stops? But I swear I tried that when the issue occurred and it didn't resolve...
> 
> Here are my logs... [Ryujinx_1.1.1364_2024-08-02_17-47-32.zip](https://github.com/user-attachments/files/16468987/Ryujinx_1.1.1364_2024-08-02_17-47-32.zip)

Hmm no after switch to OpenAL it still crackles... when going to Flurrys house...

**Comment by OmniiOMEGA** [comment]
> > > In my experience, going through loading zones causes the audio to slightly drift. This effect accumulates pretty fast and results in a pretty jarring experience.
> > > Update: Upon further testing, this only occurs on SDL2. OpenAL works just fine. SoundIO just crashes immediately and cuts my entire audio for a few seconds.
> > > ttyd-sync.mp4
> > > ttyd-desync.mp4
> > > The only log entries that I could find all looked like this:
> > > ```
> > > Mai 27 13:19:22 batpc-arch plasmashell[5681]: 00:03:05.603 |E| HLE.OsThread.4 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 17157 took more than 1000ms, resuming execution...
> > > Mai 27 13:19:45 batpc-arch plasmashell[5681]: 00:03:28.183 |W| HLE.OsThread.4 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Config:
> > > 
> > > * Ryujinx 1.1.1325+c41fddd25e8ddd3cf0b3cefeaf6595d2e4ede0fa
> > > * Vulkan
> > > * SDL2
> > > * Mods
> > >   
> > >   * Level of Detail
> > >   * Disable AA
> > > 
> > > Specs:
> > > 
> > > * Arch Linux
> > > * KDE Plasma (Wayland)
> > > * Using Pipewire + Wireplumber for audio
> > > * NVIDIA RTX 2070 (555.42.02)
> > > * Ryzen 7 2700X
> > 
> > 
> > Same issue here... my audio issues are crackle when going into paper modes... After switching audio from SDL2 to OpenAL it appears the crackling stops? But I swear I tried that when the issue occurred and it didn't resolve...
> > Here are my logs... [Ryujinx_1.1.1364_2024-08-02_17-47-32.zip](https://github.com/user-attachments/files/16468987/Ryujinx_1.1.1364_2024-08-02_17-47-32.zip)
> 
> Hmm no after switch to OpenAL it still crackles... when going to Flurrys house...

If I make the windows medium sized the crackling stops which is weird?

**Comment by OmniiOMEGA** [comment]
> I understand that increasing resolution may create unintended effects but this is a weird graphical glitch that fixes itself after using the sheet ability. This does not happen at native resolution, only once resolution is scaled up.
> 
> ![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-33](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/44de47d4-db9f-4029-98d0-cafd8fa03efd)
> 
> ![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-42](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/71be368d-1c29-4f15-b628-00f01865b62a)
> 
> ![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-53](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/f22638af-2f33-425e-b216-44cd8854a6cf)
> 
> ![Paper Mario_ The Thousand-Year Door_2024-06-05_15-54-59](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5216862/d297d18c-eded-48e3-acc4-686ac4e509c8)
> 
> [Ryujinx_1.1.1330_2024-06-05_15-50-58.log](https://github.com/user-attachments/files/15595944/Ryujinx_1.1.1330_2024-06-05_15-50-58.log)
> 
> 
> 
> Hardware:
> 
> CPU - AMD 5800X3D
> 
> GPU - NVIDIA 3080 12GB, driver 551.52
> 
> RAM - 32GB DDR4
> 
> Ryujinx version - 1.1.1330
> 
> Video renderer - Vulkan
> 
> No mods, no cheats despite what the log claims.

I think this is my issue with the resolution thing but for audio crackling

**Comment by OmniiOMEGA** [comment]
Yeah it is!! You have to go back to Native resolution? 

So we can't upscale to 2K??
